extensions/wikia/CuratedContent/CuratedContentSpecialController.class.php

```diff
@@ -187,65 +160,38 @@ public function save() {
 		}
```

Comment:
line 157: you use $this->wg->User, could you do global $wgUser so it is unified with rest of the code